rxsmartlock/src/main/java/com/freeletics/rxsmartlock/SmartLockComponent.kt

```diff
@@ -128,6 +135,8 @@ internal class SmartLockComponent {
                 .setResultCallback { result ->
                     processSaveCredentialsCallback(googleApiClient, result, username, emitter)
                 }
+        }.doOnError {
+            Timber.e(it)
```

Comment:
Do we need logging here? Or was if forgotten? Because you removed logging from other places

Reply:
I removed `onErrorSaveCredentials` and `onErrorCredentialsRequest` private method that had `Timber.e` call and replace it with this `doOnError` call. Custom exception contains same messages that was used in previous implementation.